woodwork/utils.py

```diff
@@ -14,11 +14,13 @@
     'csv': pd.read_csv,
     'text/csv': pd.read_csv,
     'parquet': pd.read_parquet,
-    'application/parquet': pd.read_parquet
+    'application/parquet': pd.read_parquet,
+    'arrow': pd.read_feather,
+    'application/arrow': pd.read_feather
```

Comment:
Since according the [FAQ](http://arrow.apache.org/faq/) the Arrow and Feather formats are identical as of Feather V2, do we want to include Feather here as well and just treat it the same as an Arrow file?

Reply:
Thats a good point. I'll add the feather extension